docs/topics/execute.adoc

```diff
@@ -68,6 +68,49 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 $ __RHAMT_HOME__/bin/rhamt-cli -Dwindup.decompiler=procyon --input __INPUT_ARCHIVE_OR_DIRECTORY__ --output __OUTPUT_REPORT_DIRECTORY__ --target __TARGET_TECHNOLOGY__ --packages __PACKAGE_1__ __PACKAGE_2__
 ----
 
+[discrete]
+[[cli_bash_completion]]
+== {ProductShortName} {CLINameTitle} Bash Completion
+
+The {ProductShortName} {CLIName} includes a set of bash completions, allowing the xref:command_line_arguments[{ProductShortName} Command-line Arguments] to be auto completed by pressing Tab when entering the commands. For instance, entering the following displays a list of available arguments.
```

Comment:
You may want to make it clearer in this intro that you have to enable bash completion for the auto complete. The way it reads to me right now, I think it implies that it includes bash completion by default.

Could maybe say "The {ProductShortName} {CLIName} provides an option to enable bash completion...".  Or perhaps later on, "For instance, when bash completion is xref:anchorname[enabled], entering..."

Reply:
I've updated this sentence in both places to indicate that this must be enabled.